be/CMakeLists.txt

```diff
@@ -963,7 +965,8 @@ if (NOT ("${MAKE_TEST}" STREQUAL "ON" AND "${BUILD_FOR_SANITIZE}" STREQUAL "ON")
 endif()
 
 set(STARROCKS_LINK_LIBS ${STARROCKS_LINK_LIBS}
-    -lresolv -lbfd -liberty -lc -lm -ldl -rdynamic -pthread -Wl,-wrap=__cxa_throw
+    ${WL_LINK_STATIC} -lbfd
+    ${WL_LINK_DYNAMIC} -lresolv -liberty -lc -lm -ldl -rdynamic -pthread -Wl,-wrap=__cxa_throw
```

Comment:
@rohitrs1983 can you please check if `iberty` can be linked statically as well? I didn't see iberty in the list of `ldd`, but  just want to make sure it is static linked.

Reply:
ok. looks like only static link library is provided by the libiberty-dev/binutils-devel